tests/e2e/runners/firefox/firefox70.bs.runner.js

```diff
@@ -33,7 +34,8 @@ async function loadFirefoxDriver () {
 const driver_gutenberg_fx = await loadFirefoxDriver();
 
 // Run test in SW mode only
-console.log('\x1b[33m%s\x1b[0m', 'Running Gutenberg tests in ServiceWorker mode only for this browser version');
+console.log('\x1b[33m%s\x1b[0m', 'Running Gutenberg and Tonedear tests in ServiceWorker mode only for this browser version');
 console.log(' ');
 
 await gutenbergRo.runTests(driver_gutenberg_fx, ['serviceworker']);
+await tonedear.runTests(driver_gutenberg_fx, ['serviceworker']);
```

Comment:
@THEBOSS0369  You can't re-use the same driver, as it is quit once the first test finishes. You need (above) to load with `const driver_tonedear_fx = await...` above, and use that driver here. This is why tests are failing with this runner. Check your other runners that you're not trying to re-use the same driver, or else those tests will also fail.

Reply:
@Jaifroid Done sir, i re used the same driver in this file only. now the test should pass without any problem